.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+go:
+  - 1.4.2
+  - 1.4.1
+  - 1.3.3
```

Comment:
I wasn't sure if just `1.4` would test each of the 1.4 builds. I also didn't add tip because we probably want to define the versions it's actually expected to build and pass on.